examples/image_streamer/artifact_bundles.py

```diff
@@ -0,0 +1,155 @@
+
+artifact_bundles_information = {
+    "name": "RHEL-7.2-artifact-bundle",
+    "id": "78836581-2b6f-4e26-9969-5667fb5837b4",
+    "id_backup": "744587b5-d739-4575-9745-5bb9b3d19f71",
+    "deployment_groups": "00c1344d-e4dd-43c3-a733-1664e159a36f"
+}
+
+artifact_bundles_to_be_created = {
+    "name": "Artifact Bundles Test",
+    "description": "Description of Artifact Bundles Test",
+    "buildPlans": [
+        {
+            "resourceUri": "/rest/build-plans/bc2135ab-e8cf-4847-9bac-1c957073cfc2",
+            "readOnly": "false"
+        }
+    ]
+}
+
+local_image_file_name = '~/HPE-ImageStreamer-Developer-2016-09-12.zip'
+destination_file_path = '~/file.zip'
+
+# Get all Artifacts Bundle
+print("\nGet all Artifact Bundles")
+artifact_bundles = image_streamer_client.artifact_bundles.get_all()
+for artifacts_bundle in artifact_bundles:
+    pprint(artifacts_bundle)
+
+
+# Create an Artifact Bundle
+print("\nCreate an Artifact Bundle")
+response = image_streamer_client.artifact_bundles.create(artifact_bundles_to_be_created)
+pprint(response)
+
+
+# Get the Artifacts Bundle by ID
+print("\nGet the Artifact Bundles by ID")
+artifacts_bundle = image_streamer_client.artifact_bundles.get(artifact_bundles_information['id'])
+pprint(artifacts_bundle)
+
+
+# Get the Artifacts Bundle by Name
+print("\nGet the Artifact Bundles by Name")
+artifacts_bundle = image_streamer_client.artifact_bundles.get_by_name(artifact_bundles_information['name'])
+pprint(artifacts_bundle)
+
+
+# Update an Artifact Bundle
+print("\nUpdate an Artifact Bundle")
+artifact_bundle = image_streamer_client.artifact_bundles.get_by_name(artifact_bundles_to_be_created['name'])
+response = image_streamer_client.artifact_bundles.update(artifact_bundle, 'Artifact Bundles Test Updated')
+pprint(response)
+
+
+# Delete an Artifact Bundle
+print("\nDelete an Artifact Bundle")
+artifact_bundle = image_streamer_client.artifact_bundles.get_by_name(artifact_bundles_to_be_created['name'])
+image_streamer_client.artifact_bundles.delete(artifact_bundle)
+
+
+# Get all Backups for Artifact Bundles
+print("\nGet all Backups for Artifact Bundles")
+artifacts_bundle = image_streamer_client.artifact_bundles.get_backups()
+pprint(artifacts_bundle)
+
+
+# Get Backup for an Artifacts Bundle
+print("\nGet Backups for an Artifacts Bundle")
+artifacts_bundle = image_streamer_client.artifact_bundles.get_backups_by_id(artifact_bundles_information['id_backup'])
+pprint(artifacts_bundle)
+
+
+# Download the archive of the Artifact Bundle
+print("\nDownload the archive of the Artifact Bundle")
+artifacts_bundle = image_streamer_client.artifact_bundles\
+    .download_archive_artifact_bundle(artifact_bundles_information['id'], destination_file_path)
+pprint(artifacts_bundle)
+
+
+# Download the Artifact Bundle
+print("\nDownload the Artifact Bundle")
+artifacts_bundle = image_streamer_client.artifact_bundles.download_artifact_bundle("0ABDE00534F", destination_file_path)
```

Comment:
Reading the example, without check the method implementation or the documentation, I can't guess what 0ABDE00534F means.
Could you refactor and extract it to a variable, preferently located in the top of the file, near the defined variables artifact_bundles_information and artifact_bundles_to_create?